jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildContext.java

```diff
@@ -97,6 +98,15 @@ public Builder setBaseImageConfiguration(ImageConfiguration imageConfiguration)
       return this;
     }
 
+    /**
+     * Enables/Disables the platform tags.
```

Comment:
Sets whether to automatically add architecture suffix to tags for platform-specific images when building multi-platform images. For example, when building amd64 and arm64 images for a given tag, the final tags will be `<tag>-amd64` and `<tag>-arm64`.